.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ workflows:
       - test-node10
       - test-node12
   test_and_publish:
+    context: npm
```

Comment:
The context goes on the job instance in the workflow:
```
workflows:
  version: 2
  my-workflow:
    jobs:
      - run-tests:
          context: org-global
```

Reply:
🤦‍♂ updating